### PR TITLE
dgram: test setting/getting socket buffer size

### PR DESCRIPTION
**Pull request by Loghorn**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://nodejs.org/dist/latest-v8.x/docs/api/dgram.html
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/node/index.d.ts [hunk 1]
```diff
@@ -2695,6 +2697,10 @@ declare module "dgram" {
         setMulticastLoopback(flag: boolean): void;
         addMembership(multicastAddress: string, multicastInterface?: string): void;
         dropMembership(multicastAddress: string, multicastInterface?: string): void;
+        getRecvBufferSize(): number;
+        getSendBufferSize(): number;
+        setRecvBufferSize(size: number): void;
+        setSendBufferSize(size: number): void;
```
Comment (Flarna) [inline types/node/index.d.ts]: These APIs are already present (see #20535).

**Review by Flarna — CHANGES_REQUESTED** [review]

File: types/node/index.d.ts (hunk 1 above)
Comment (Loghorn) [reply types/node/index.d.ts]: ok, this proves I can't read or use grep ☹️

Comment (Flarna) [reply types/node/index.d.ts]: Or git merges without thinking too much. The PR I referred to was merged last night.

File: types/node/node-tests.ts [hunk 2]
```diff
@@ -1365,6 +1365,10 @@ namespace dgram_tests {
         });
         ds.send(new Buffer("hello"), 5000, "127.0.0.1");
         ds.setMulticastInterface("127.0.0.1");
+        ds.getRecvBufferSize();
```
Comment (Flarna) [inline types/node/node-tests.ts]: Tests are also present around 100 lines below :smile:

Comment (Loghorn) [reply types/node/node-tests.ts]: Ok, I close this PR and close myself in the bathroom to cry 😢 😢 😢 😢 😢 😢

**Pull request closed without merging**